vscode-sbcl-clpm.md

```diff
@@ -0,0 +1,472 @@
+  [macports](https://www.macports.org/),
+  [asdf-vm](https://github.com/smashedtoatoms/asdf-sbcl), etc.
+- **CLPM** You can install CLPM by [following the
+  instructions](https://www.clpm.dev/#installing).  If you want to skip
+  a few steps, you can download the binary from that site, drop it in
+  your path, and run
+  ```sh
+  curl -L https://smashedtoatoms.com/rando/configure-clpm-to-work-with-sbcl.sh | bash
+  ```
+  To uninstall, delete it from your path and run
+  ```sh
+  curl -L https://smashedtoatoms.com/rando/configure-clpm-to-work-with-sbcl.sh | bash -s -- cleanup
+  ```
+  If running shell scripts off the internet gives you the creeps, just
+  [follow the instructions](https://www.clpm.dev/#installing)
+- **rlwrap** You can install rlwrap using your OS package manager
+
+### The TL;DR; Version
+
+_If you get stuck, switch to the non-tldr version.  Sometimes knowing
+what it is doing helps._
+
+1. Create a folder for your new project
+   1. Start a repl
+      ```bash
+      rlwrap sbcl
+      ```
+   2. Create a directory for the project called `experiment`
+      ~~~lisp
+      (ensure-directories-exist "experiment/") ;; That last slash matters
+      ~~~
+   3. Exit the repl
+      ~~~lisp
+      (sb-ext:exit) ;; You can also just do a ctrl-d
+      ~~~
+2. Open vscode in your newly created directory
+   ```bash
+   cd experiment
+   code .
+   ```
+3. Create a new Common Lisp System using Alive.
+   1. Inside of vscode, Open Command Palette on the menu at the top
+      `View/Command Palette`
+   2. Generate skeleton: `Alive: System Skeleton`
+   3. The previous command should have generated the following directory
+      structure
+      - src
+        - app.lisp
+           ~~~lisp
+           (defpackage :app
+             (:use :cl))
+
+           (in-package :app)
+           ~~~
+      - test
+        - all.lisp
+           ~~~lisp
+           (defpackage :test/all
+             (:use :cl
+                   :app)
+             (:export :test-suite))
+
+           (in-package :test/all)
+
+           (defun test-suite ()
+             (format T "Test Suite~%"))
+           ~~~
+      - experiment.asd
+         ~~~lisp
+         (in-package :asdf-user)
+
+         (defsystem "experiment"
+           :class :package-inferred-system
+           :depends-on ("experiment/src/app")
+           :description ""
+           :in-order-to ((test-op (load-op "experiment/test/all")))
+           :perform (test-op (o c) (symbol-call :test/all :test-suite)))
+
+         (defsystem "experiment/test"
+           :depends-on ("experiment/test/all"))
+
+         (register-system-packages "experiment/src/app" '(:app))
+         (register-system-packages "experiment/test/all" '(:test/all))
+         ~~~
+4. Set up a clpm bundle in your terminal
+   1. Start up your lisp repl
+      ```bash
+      rlwrap sbcl
+      ```
+   2. Init/install your clpm bundle, switch to your project context, and
```

Comment:
Did you say what it is to init the clpm bundle?

Reply:
Yea, that is the `bundle-init `call immediately following this.